lib/repository_client/cli.rb

```diff
@@ -3,11 +3,36 @@
 module RepositoryClient
   # The command line interface
   module CLI
+    # rubocop:disable Metrics/MethodLength
     def self.start(command, options)
       case command
       when 'deposit'
         RepositoryClient::Deposit.run(options)
+      when 'help'
+        puts <<~HELP
+          DESCRIPTION:
+            The SDR Command Line Interface is a tool to interact with the Stanford Digital Repository.
+
+          SYNOPSIS:
+            sdr [options] <command>
+
+          OPTIONS:
+            --host (string)
```

Comment:
this should be `service-url`: e.g.: `--service-url https://sdr-api-stage.stanford.edu`

Reply:
Done.